regression/contracts/assigns_enforce_elvis_5/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--enforce-contract foo
+^EXIT=1$
+^SIGNAL=0$
+^main.c:\d+:\d+: error: illegal target in assigns clause$
+^CONVERSION ERROR$
+--
+--
+Check that Elvis operator expressions '(cond ? *if_true : *if_false)' are rejected from assigns clauses.
```

Comment:
Can we add a comment here that this is consistent with major compilers? (Perhaps also check the C standard?)
Because it's not super intuitive why this should be rejected and not `*(cond ? if_true : if_false)` ...

Reply:
I guess @feliperodri has. found the explanation, so he could add a comment now.